data_generators/src/generators.rs

```diff
@@ -0,0 +1,15 @@
+mod block_states;
+mod blocks;
+mod entities;
+mod inventory;
+mod items;
+mod simplified_block;
+
+pub fn generate_all() {
```

Comment:
Yeah, I think this pr should've been split into many PRs for worlds, dimensions, generators, 1.18, etc., but I was updating it like "old dimension_codec.nbt doesn't work, then I need to implement multidimension and biome loading, the packet doesn't work - rewrite proxy, a packet couldn't be deserialized - fix a bug in hematite-nbt" and so on, and I didn't want to stop if something doesn't work. I'm not sure if this would be compatible with old generators, but I think I can try to move it.

Reply:
I think it's better to leave generators in this PR because:
1) 1.16 requires a biome generator, which is not implemented here (it extracts biome information at runtime rather than hardcode the `Biome` enum in generators) and I don't see any solutions other than "write a generator and then delete it" or "it doesn't work, but it'll work when 1.18 is merged" and have CI failing.
2) `main` doesn't have `feather_base::consts`, and this pr won't be able to use `data_generators::extract_vanilla_data`, so we'll need to have duplicate code across 2 PRs and then create another pr to remove it (or forget about it and have 2 different implementations, like we had `feather-blocks` and `libcraft-blocks`)
3) minecraft-data has removed `Item::Air` from their 1.16 data, but `main` still uses it as a default item, that'll create even more problems.
4) Should I replace `libcraft-blocks` with `feather-blocks` in this PR or in generators PR? Both variants have obvious issues such as merge conflicts in `use` statements.